gh-136882: Update stale link in the basic logging tutorial. (#136885)
python/cpython: Doc/howto/logging.rst

favourite beverage and carry on.

If your logging needs are simple, then use the above examples to incorporate
logging into your own scripts, and if you run into problems or don't understand

> something, please post a question using the "New Topic" button on the `Python

Reviewer: ```suggestion
something, please post a question in the Help category of the `Python
```
I do not think there is need to point out the button (it is self explanatory IMO, and if we really want to make it simple then we could directly link to the create a topic page and cut out the [middleman] button), we can highlight the category instead?

Author: The reason I mentioned the button label is that "post a question" and "Create Topic" do not necessarily automatically mean the same thing to a newcomer, though it might seem self-explanatory to those familiar with the forum - newcomers might land on the page and look around for a "Post question" button or link, for example. The link automatically goes to the help category page which says "About the Python help category" and they don't necessarily care when reading this bit exactly which part of the discussion forum they'll be taken to. Once they get there, they'll see there's a Help category and what its scope is, and know from the foregoing which button to look for to post their question.

Reviewer 2: If they're a newcomer, they won't see "New Topic" until after they've signed up.

Author: Ah, OK, fair enough - I hadn't thought of that.
